app/views/match_mailer/match_date_changed.html.erb

```diff
@@ -1,7 +1,15 @@
 <h1><%= t('mailer.hello', firstname: @user.first_name) %></h1>
-<p>
-  <%= t('match_mailer.next_game', name: @match.event.name, team_home: @match.team_home.name, team_away: @match.team_away.name) %>
-  <%= t('.info', start_time: I18n.localize(@match.start_time)) %>
-</p>
-<p><%= t('mailer.have_fun') %></p>
-<%= link_to @match.event.name, @match.event %>
\ No newline at end of file
+<p><%= t('.info_html',
+         team_link: link_to(@opponent_team.name, team_url(@opponent_team)),
+         event_link: link_to(@match.event.name, event_url(@match.event))) %></p>
+<% if @match.start_time %>
+  <p><%= t('match_mailer.start_time',
```

Comment:
here you seem to have indented; we should stay consistent, I think

Reply:
Please check/test in browser - you might want to indent in html because of language conventions. In txt this will be a hardcoded indentation, that is not desired.